src/dotnet/CarbonAware.Plugins.NewBasicJsonPlugin/CarbonAwareNewBasicJsonPlugin.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
+using CarbonAware.Model;
+using Microsoft.Extensions.Logging;
+
+namespace CarbonAware.Plugins.NewBasicJsonPlugin;
+
+public class CarbonAwareNewBasicJsonPlugin : ICarbonAware
+{
+    private readonly ILogger<CarbonAwareNewBasicJsonPlugin> Logger;
+
+    public CarbonAwareNewBasicJsonPlugin(ILogger<CarbonAwareNewBasicJsonPlugin> logger)
+    {
+        Logger = logger ?? throw new ArgumentNullException(nameof(logger));
+    }
+
+    public async Task<IEnumerable<EmissionsData>> GetEmissionsDataAsync(IDictionary props)
+    {
+        return await Task.Run(() => FakeData());
+    }
+
+    public async Task<IEnumerable<EmissionsData>> GetEmissionsDataAsync(IDictionary props, Func<QueryObject, bool> filter)
+    {
+        Logger.LogInformation("New Data");
+        foreach (var k in props.Keys)
+        {
+            Logger.LogInformation($"Key: {k} - Value: {props[k]}");
+        }
+        return await Task.Run(() => FakeData().Where((x) => filter(new QueryObject { 
+            Location = x.Location,
+            Time = x.Time,
+            Rating = x.Rating
+        })));
+    }
+
+    private IEnumerable<EmissionsData> FakeData()
```

Comment:
I debate if perhaps the FakeData should be in its own file for clarity and to separate the hardcoded magic strings to a single location where its apparent what they are

Reply:
@gfmatthews , as mentioned above, some more work needs to be done here.